fitbenchmarking/cost_func/poisson_cost_func.py

```diff
@@ -25,8 +25,10 @@ class PoissonCostFunc(CostFunc):
     This cost function is intended for positive values.
 
     This cost function is not a least squares problem and as such will not work
-    with least squares minimizers. Please use `algorithm_type` to select either
-    `deriv_free` or `general` solvers.
+    with least squares minimizers. Please use `algorithm_type` to select
+    `general` solvers.
+    See options docs for information on how to do this:
+    https://fitbenchmarking.readthedocs.io/en/latest/users/options/fitting_option.html#algorithm-type-algorithm-type
```

Comment:
```suggestion
    See options docs (:ref:`fitting-option`) for information on how to do this.
```

Reply:
Use the sphinx linking mechanism, rather than urls, to go to the docs, otherwise a user may be linked to the wrong version of the docs, of links may get out of date (I think it's more important that the docstring on sphinx renders correctly than the comment in the code).  

This suggestion goes to the top of the options page, not the algorithm type section you originally linked to -- I'm happy with either.